style.css

```diff
@@ -53,7 +69,29 @@ body{
     color: var(--primary-maroon);
 }
 
+.pointer{
+    cursor: pointer;
+}
+
+.background-light-grey{
+    background-color: var(--light-grey);
+}
 
+.position-static{
+    position: static !important;
```

Comment:
Generally, try avoiding important and use something with more specificity. Until unless it can't be done without !important

Reply:
Sure thing, will keep that in check